src/winston.utilities.ts

```diff
@@ -45,7 +45,14 @@ const nestLikeConsoleFormat = (
     const color = options.colors && nestLikeColorScheme[level] || ((text: string): string => text);
     const yellow = options.colors ? clc.yellow : ((text: string): string => text);
 
-    const stringifiedMeta = safeStringify(meta);
+    // Apply transformation function to meta data. If no transformation function
+    // is supplied, just remove the `value` property from the meta data.
+    // https://github.com/gremo/nest-winston/issues/626
+    const transformedMeta = options.transformMeta
+      ? options.transformMeta(meta)
+      : { ...meta, value: undefined };
```

Comment:
`const transformedMeta = options.transformMeta ? options.transformMeta(meta) : meta;` ?

Reply:
If we make this change, the `value` property will be logged when no transformation function is supplied. I intentionally made the default behavior to not log `value`.